src/saturn_engine/core/api.py

```diff
@@ -18,6 +18,18 @@ class Inventory:
     options: dict
 
 
+@dataclasses.dataclass
+class Topic:
+    name: str
+
+
+@dataclasses.dataclass
+class JobDefinition:
+    name: str
+    pipeline_info: PipelineInfo
```

Comment:
I was suggesting something like

```
name: str
template: QueueItem
minimal_interval: str
```

So a JobDefinition is something that creates QueueItem